vcf-to-gds/vcf-to-gds.wdl

```diff
@@ -65,8 +65,10 @@ task unique_variant_id {
 		# This is a workaround for the Python code to work correctly
 		# Symlinks would be preferable, but they do not work on GCS
```

Comment:
Can we delete this now?

Reply:
The comment needs to be modified to be more accurate, but the tl;dr is that the workaround is still needed here.

Reply:
Agreed. It seems like you mostly need to get rid of the part which says "they do not work on GCS".